packages/web-client/src/webdav/index.ts

```diff
@@ -37,7 +37,7 @@ export const webdav = (options: WebDavOptions): WebDAV => {
   const listFilesByIdFactory = ListFilesByIdFactory(dav, options)
   const { listFilesById } = listFilesByIdFactory
 
-  const getFileInfoFactory = GetFileInfoFactory(listFilesFactory, options)
+  const getFileInfoFactory = GetFileInfoFactory(listFilesFactory)
```

Comment:
Please keep passing the `options` to the factories. We don't always need them now, but we might in the future. That way, all factories have the same options.